src/buybutton.js

```diff
@@ -1,16 +1,18 @@
-import ShopifyBuy from 'shopify-buy/polyfilled';
+// import ShopifyBuy from 'shopify-buy/polyfilled';
+import ShopifyBuy, {Config} from 'shopify-buy';
 import UI from './ui';
 import productTemplates from './templates/product';
 
 window.ShopifyBuy = window.ShopifyBuy || ShopifyBuy;
+window.Config = window.Config || Config;
 
 ShopifyBuy.UI = window.ShopifyBuy.UI || {
   domains: {},
 
-  init(client, integrations = {}, styleOverrides) {
-    const domain = client.config.domain;
+  init(client, config, integrations = {}, styleOverrides) {
```

Comment:
it used to exist on the client, but doesn't with `1.0.0` - i would've preferred to keep it the same too. @swalkinshaw @minasmart thoughts?